doc/source/whatsnew/v0.24.0.txt

```diff
@@ -147,7 +147,7 @@ MultiIndex
 I/O
 ^^^
 
--
+- (Only for Windows) Bug in :meth:`DataFrame.to_csv`, in which all `\n`s are converted to `\r\n` (:issue:`20353`)
```

Comment:
I would move the Windows portion as follows:

`Bug in :meth:`DataFrame.to_csv`, in which all `\n`s are converted to `\r\n` on Windows (:issue:`20353`)`